### ISSUE TITLE
MultiAction popovers not accessible using a screen reader

### ISSUE THREAD

**Issue by Arkalex** [Sage/carbon#7032]
### Description

Given a user using a screen reader
When the user tries to navigate through the options in a MultiAction component
Then he is not able to do it

### Reproduction

https://carbon.sage.com/?path=/docs/multi-action-button--docs

### Steps to reproduce

1. Activate screen reader (cmd + F5 for MacOS)
2. Try to navigate to the actions (ctrl + option + left/right arrows)

### JIRA ticket numbers (Sage only)

SBS-112819

### Carbon version

144.0.0

### Relevant browsers

Chrome

### Relevant OSs

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.
